### PR TITLE
Bug fix: update GHES migration procedure reusable

### PR DESCRIPTION
**Pull request by ethanpalm**
### Review summary

This PR updates a reusable to link to the correct step in preparing to migrate data to a GHES instance.

`ready-to-import-migrations.md` links to "[Migrating data to your enterprise](https://docs.github.com/en/enterprise-server@2.22/admin/user-management/migrating-data-to-your-enterprise)" when it should link to "[Preparing to migrate data to your enterprise](https://docs.github.com/en/enterprise-server@2.22/admin/user-management/preparing-to-migrate-data-to-your-enterprise)."

The reusable is currently used in "[Exporting migration data from your enterprise](https://docs.github.com/en/enterprise-server@2.22/admin/user-management/exporting-migration-data-from-your-enterprise)" and "[Exporting migration data from GitHub.com](https://docs.github.com/en/enterprise-server@2.22/admin/user-management/exporting-migration-data-from-githubcom)."

Closes github/docs-content#3091

### Changes in this PR

| Article in staging | Source | Changes |
| --- | --- | --- |
`ready-to-import-migrations.md` | [Source](data/reusables/enterprise_migrations/ready-to-import-migrations.md) | Updates link to [Preparing to migrate data to your enterprise](https://docs.github.com/en/enterprise-server@2.22/admin/user-management/preparing-to-migrate-data-to-your-enterprise)

### Versioning

All GHES

### Tests
✅

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks for opening a PR @ethanpalm 🎉 I'll triage this for review ⚡

File: data/reusables/enterprise_migrations/ready-to-import-migrations.md [hunk 1]
```diff
@@ -1 +1 @@
-1. To import the archived migration data to a {% data variables.product.prodname_ghe_server %} instance, see "[Importing migration data to {% data variables.product.prodname_ghe_server %}](/enterprise/admin/guides/migrations/importing-migration-data-to-github-enterprise-server/)".
+1. To import the archived migration data to a {% data variables.product.prodname_ghe_server %} instance, see "[Preparing to migrate data to your enterprise](/enterprise/admin/user-management/preparing-to-migrate-data-to-your-enterprise)".
```
Comment (felicitymay) [inline data/reusables/enterprise_migrations/ready-to-import-migrations.md:1]: Hi @ethanpalm 👋🏻 

Thank you for your contribution 😄 

Looking at [Exporting migration data from GitHub.com - GitHub Docs](https://docs-2930--bugfix-ghes-migrati.herokuapp.com/en/enterprise-server@2.22/admin/user-management/exporting-migration-data-from-githubcom#generating-a-migration-archive), I can see that your PR nicely fixes the reported issue.

I think that we might want to go slightly further and also update the text in the step to emphasize the need to prepare the data before importing it. How about:

```suggestion
1. To prepare the archived migration data for import into a {% data variables.product.prodname_ghe_server %} instance, see "[Preparing to migrate data to your enterprise](/enterprise/admin/user-management/preparing-to-migrate-data-to-your-enterprise)".
```

**Review by felicitymay — APPROVED** [review]
Thanks for accepting the suggestion. It'll be great to get this merged and link people to the best topic.

**Review by teetee-make — COMMENTED** [review]
Good

**Pull request merged**